src/components/JaegerToolbar/LookBack.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Col, Form, FormGroup, FormControl, FieldLevelHelp } from 'patternfly-react';
+import { KialiAppState } from '../../store/Store';
+import ToolbarDropdown from '../../components/ToolbarDropdown/ToolbarDropdown';
+import { JaegerActions } from '../../actions/JaegerActions';
+import { ThunkDispatch } from 'redux-thunk';
+import { KialiAppAction } from '../../actions/KialiAppAction';
+
+interface LookBackProps {
+  fetching: boolean;
+  setLookback: (lookback: string) => void;
+  lookback: string;
+  onChangeCustom: (when: string, dateField: string, timeField: string) => void;
+}
+
+export class LookBack extends React.PureComponent<LookBackProps, {}> {
+  lookBackOptions = {
```

Comment:
The labels conventions here differ with what we have in similar dropdowns (it's fine IMO to have different values, but the names should be consistent) See in `Config.ts` : we capitalize only the first word ("Last 5 min" , "Last hour" ...). Also we have "Last day" instead of "Last 24 hours".

Reply:
I set the same value that in Jaeger UI Toolbar, I don't see any problem to format like the others dropdowns

Reply:
Why this wasn't done before merge?

Reply:
Let me fix that in https://github.com/kiali/kiali-ui/pull/1008